modules/deployments-comparing-deploymentconfigs.adoc

```diff
@@ -5,25 +5,33 @@
 [id='deployments-comparing-deploymentconfigs-{context}']
 = Comparing Deployments and DeploymentConfigs
 
-Because deployment configurations existed in {product-title} prior to
-deployments being added in Kubernetes 1.2, the latter object type naturally
-diverges slightly from the former. The long-term goal in {product-title} is to reach
-full feature parity in Deployments and switch to using them as a
-single object type that provides fine-grained management over applications.
-
-Deployments are supported to ensure upstream projects and examples that use the
-new object type can run smoothly on {product-title}. Given the current feature
-set of Deployments, you might want to use them instead of deployment
-configurations in {product-title} if you do not plan to use any of the following
-in particular:
-
-- Image streams
-- Lifecycle hooks
-- Custom deployment strategies
-
-The following sections go into more details on the differences between the two
-object types to further help you decide when you might want to use Kubernetes
-deployments over deployment configurations.
+Both Kubernetes Deployments and OpenShift Container Platform-provided
+DeploymentConfigs are supported in OpenShift Container Platform; however, it is
+recommended to use Deployments unless you need a specific feature or behavior
+provided by DeploymentConfigs.
+
+The following sections go into more detail on the differences between the two
+object types to further help you decide which type to use.
+
+[id='deployments-design-{context}']
+== Design
+
+One important difference between Deployments and DeploymentConfigs is the
```

Comment:
@tnozicka Can you see if my attempt here at describing these situations is accurate, or if not, what specifically needs to be changed? Thanks!